security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -17,6 +17,72 @@ To access the latest release, see xref:../../security/compliance_operator/co-man
+
+* Prior to this release, Compliance Operator STIG control CNTR-OS-000720 selected rule 'audit-rules-suid-privilege-function' but the rule was not available in CO, therefore no output was generated. With this release, rule 'audit-rules-suid-privilege-function' is now available in Compliance operator and listed in the scan output. For more information, see (link:https://issues.redhat.com/browse/CMP-3558[CMP-3558]).
+
+* In previous versions of Compliance Operator, scanning with the `ocp4-stig` profile would fail for the rule `ocp4-stig-modified-audit-log-forwarding-uses-tls` even if TLS is enabled correctly. This would occur because the `tls://` field is no longer required by the `ClusterLogForwarder` resource, causing the scan output to show an incorrect `FAIL` result. With this release, the protocol prefix is not required and the scan output produces correct results. 
+// For more information, see (link:https://issues.redhat.com/browse/CMP-3566[CMP-3566]).
+
+* Previously, a control value called `unsupportedConfigOverrides` allowed users to opt into unsupported behavior. This is a bad practice because it allows OpenShift administrators to opt into configuration overrides which may produce non-compliant scan results from Compliance Operator. With this release, Compliance Operator no longer uses the `unsupportedConfigOverrides` control value. 
+// For more information, see (link:https://issues.redhat.com/browse/CMP-3580[CMP-3580]).
+
+* For prior releases of Compliance Operator, `resource-requests-limits` rules were missing a variable reference and produced output on that basis. With this release, the variable reference is available to `resource-requests-limits` rules and the erroneous output is eliminated. For more information, see (link:https://issues.redhat.com/browse/CMP-3582[CMP-3582]).
+
+* Previously, the `ocp4-routes-rate-limit` rule required setting rate limits for all routes outside the `openshift` and `kube` namespaces. However, using the feature and scanning for it presented problems because other namespaces managed by critical operators should not be modified and not be scanned for the modification by Compliance Operator. With this release, routes managed by critical operators are not flagged as errors by the Compliance Operator. 
+// For more information, see (link:https://issues.redhat.com/browse/CMP-3589[CMP-3589]).
+
+* In prior versions of Compliance Operator, an update of the network stack from Openshift-SDN to OVNKubernetes would trigger an alert from a complianceScan Instance for `sdn not found`. In this release, Compliance Operator has been updated to stop scanning for OpenShift-SDN when it has been replaced by OVNKubernetes. For more information, see (link:https://issues.redhat.com/browse/CMP-3591[CMP-3591]).
```

Comment:
There are slightly errors: "stop scanning" is a bit confusing:
How about 
In prior versions of Compliance Operator, a compliancescan will report a warning `sdn not found` when openshift-sdn was not found(such as when using OVNKubernetes instead). In this release, Compliance Operator suppress the warning when openshift-sdn is not the active networking provider.